tutorials/quick_start/main.py

```diff
@@ -50,6 +50,8 @@ def argument_handler():
                         help='Run according to a list of models and parameters taken from a csv file')
     parser.add_argument('--validation_set_limit', type=int, default=None,
                         help='Limits the number of images taken for evaluation')
+    parser.add_argument('--export_model', action="store_true",
```

Comment:
where will it be exported to? maybe we should also add an argument for path to save the model?

Reply:
but where is the model being saved? in which folder?

Reply:
local folder that is generated automatically. There is a print that shows the location.  This is the same in the notebook tutorials.